src/mapping.ts

```diff
@@ -112,6 +112,7 @@ export function handleNewHash(event: NewHash): void {
         [ipfsHash],
       );
     }
+    entity.dataHash = computeHash(serializeTransaction(entity));
```

Comment:
I'm pretty sure this doesn't work, but just in case, did you try?
```suggestion
    entity.dataHash = computeHash(transactions[index].toString());
```

Reply:
I did, it throws with `JSON value is not a string`